src/UserSystem/Repositories/InMemoryUserRepository.cs

```diff
@@ -0,0 +1,50 @@
+using Interfaces;
+
+namespace Repositories;
+
+public class InMemoryUserRepository : IUserRepository
```

Comment:
Please update the name of this class. It doesn't follow our name classes convention.